.travis.yml

```diff
@@ -7,6 +7,8 @@ addons:
       # python for map file splitter
       - python3
       - python3-yaml
+      # up-to-date java version
+      - oracle-java8-installer
```

Comment:
Travis is using a super old java version (e.g. 1.8.0_u30), but some JavaFX objects were introduced in u40, therefore the build fails